pkgs/tools/package-management/cde/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchgit }:
+
+stdenv.mkDerivation {
+  name = "cde-0.1";
+  src = fetchgit {
+    url = "https://github.com/pgbovine/CDE.git";
```

Comment:
Thanks for fixing - was planning on doing it tomorrow evening but you beat me to it :smile:.